march_hardware_interface/config/march4/controllers.yaml

```diff
@@ -20,3 +21,13 @@ march:
         - left_ankle
         - right_hip_aa
         - left_hip_aa
+#      gains: # Required because we're controlling an effort interface
+#        right_hip_aa: {p: 20, i: 0, d: 5, i_clamp: 100, publish_state: true, antiwindup: true} # TODO(BaCo) tune HAA joints
```

Comment:
I think it's best if the joint order for the gains is the same as the joint order defined a few lines above